docs/20-concepts/05-styling/34-theming.mdx

```diff
@@ -169,7 +169,7 @@ stateDiagram-v2
 		KoliBri-Komponenten passen, können in eigenen Komponenten-Bibliotheken aufgebaut und unabhängig gepflegt werden.
 		Hierfür bieten wir die sogenannte <strong>KoliBri-Factory</strong> (Stack) an (siehe{' '}
 		<KolLink
-			_href="https://github.com/public-ui/kolibri/blob/main/docs/ARCHITECTURE.md#erweitert"
+			_href="https://public-ui.github.io/docs/concepts/architecture#modularisierung"
```

Comment:
```suggestion
			_href="/docs/concepts/architecture#modularisierung"
```

Innerhalb der Docs bitte relative Links verwenden, damit sie z.B. auch lokal und (zukünftig) in einer Staging-Umgebung korrekt funktionieren.

Könntest du bitte auch die englische Version einmal prüfen und die versionierten Docs ggf. ebenfalls korrigieren? Die findest du hier:

- `versioned_docs/`
- `i18n/en/docusaurus-plugin-content-docs/` (current und version-* beachten)